libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionStepResources.java

```diff
@@ -98,7 +98,7 @@ private void thenStep(LegStep upcomingStep, LegStep followOnStep, double current
     thenStepManeuverType = followOnStep.maneuver().type();
     thenStepManeuverModifier = followOnStep.maneuver().modifier();
     // Should show then step if the upcoming step is less than 25 seconds
```

Comment:
This comment is now lying. This is an example of why I'm not a big fan of comments :trollface:

Reply:
> The best comment is a good name for a method or class.

or _variable_ in this case 😛
👀 Code Smells 👉 [Comments](https://refactoring.guru/smells/comments)